### PR TITLE
feat: add dubbo3 demo

### PR DESCRIPTION
**Pull request by LaurenceLiZhixin**
add dubbo3 demo in general/dubbo3
Pls don't merge until   https://github.com/apache/dubbo-go/pull/959 is merge

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -17,9 +18,12 @@ require (
...
 	github.com/uber/jaeger-lib v2.2.0+incompatible // indirect
-	google.golang.org/grpc v1.26.0
+	google.golang.org/grpc v1.34.0
+	google.golang.org/grpc/examples v0.0.0-20210107225343-0bd76be2bb6b // indirect
 )
 
 replace github.com/envoyproxy/go-control-plane => github.com/envoyproxy/go-control-plane v0.8.0
 
+replace github.com/apache/dubbo-go v1.5.5 => ../dubbo-go
+
```
Comment (LaurenceLiZhixin) [inline go.mod:28]: after dubbo3.0 merged, I 'll change it

**Comment by cityiron** [comment]
If add dubbogo3.0 demo to 3.0 branch?

**Comment by LaurenceLiZhixin** [comment]
> If add dubbogo3.0 demo to 3.0 branch?

I think ur right.
Pls add a new 3.0 branch for me.

**Comment by cityiron** [comment]
> > If add dubbogo3.0 demo to 3.0 branch?
> 
> I think ur right.
> Pls add a new 3.0 branch for me.

I changed this PR to 3.0 branch.

File: general/dubbo3/unary-client/dubbo3-client/cmd/client.go [hunk 2]
```diff
@@ -50,22 +51,73 @@ func main() {
...
+			rsp, err := grpcGreeterImpl.BigUnaryTest(ctx, &BigDataReq)
+			if err != nil{
 				panic(err)
 			}
-			gxlog.CInfo("client response result: %v\n", reply)
-			//wg.Done()
+			fmt.Println("rsp len = ", len(rsp.Data))
+			time.Sleep(time.Second)
+
+			//rsp, err = grpcGreeterImpl.BigUnaryTest(context.Background(), &BigDataReq)
```
Comment (AlexStocks) [inline general/dubbo3/unary-client/dubbo3-client/cmd/client.go:78]: If u do not  want to use the following codes, pls delete them.

File: general/dubbo3/unary-client/grpc-client/main.go [hunk 3]
```diff
@@ -42,19 +41,66 @@ func main() {
...
-			if err != nil {
-				log.Fatalf("could not greet: %v", err)
+			// Test Big Unary
+			rsp, err := c.BigUnaryTest(context.Background(), &BigDataReq)
+			if err != nil{
+				panic(err)
 			}
-			log.Printf("####### get client %+v", r)
+			fmt.Println("rsp len = ", len(rsp.Data))
+			////
```
Comment (AlexStocks) [inline general/dubbo3/unary-client/grpc-client/main.go:63]: delete them.

File: go.sum [hunk 4]
```diff
@@ -100,13 +100,19 @@ github.com/alecthomas/units v0.0.0-20190717042225-c3de453c63f4/go.mod h1:ybxpYRF
 github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d/go.mod h1:rBZYJk541a8SKzHPHnH3zbiI+7dagKZ0cgpgrD7Fyho=
 github.com/alibaba/sentinel-golang v0.6.2 h1:1OjjpljJbNKWp9p5RJKxOqS1gHGZPUWPlCcokv5xYJs=
 github.com/alibaba/sentinel-golang v0.6.2/go.mod h1:5jemKdyCQCKVf+quEia53fo9a17OSe+wnl9HX2NbNpc=
+github.com/alibaba/sentinel-golang v1.0.1 h1:WlhN0XUxRyfkiDc8TO6CcRrnakwFP9zFtvJTYxZRCgI=
```
Comment (AlexStocks) [inline go.sum:103]: using 'go mod tidy' to delete useless old version sentinel-golang.

File: general/dubbo3/server/dubbo3-server/conf/server.yml [hunk 5]
```diff
@@ -4,17 +4,16 @@
 application:
   organization: "dubbo.io"
   name: "GreeterGrpcServer"
-  module: "dubbo-go greeter grpc server"
+  module: "dubbo-go3.0 greeter server"
```
Comment (AlexStocks) [inline general/dubbo3/server/dubbo3-server/conf/server.yml:7]: dubbo-go v3

File: general/dubbo3/stream-client/dubbo3-client/conf/client.yml [hunk 6]
```diff
@@ -10,26 +10,24 @@ connect_timeout: "3s"
 application:
   organization: "dubbo.io"
   name: "GreeterGrpcConsumer"
-  module: "dubbo-go greeter grpc client"
+  module: "dubbo-go3.0 greeter client"
```
Comment (AlexStocks) [inline general/dubbo3/stream-client/dubbo3-client/conf/client.yml:13]: dubbo-go v3

**Review by zouyx — COMMENTED** [review]
Make checking success thx.

**Comment by LaurenceLiZhixin** [comment]
After PR: https://github.com/apache/dubbo-go/pull/1071 
merged, the existing problems would be fixed.

**Pull request closed without merging**